models/payment/src/transaction/transaction-draft/generator.ts

```diff
@@ -0,0 +1,22 @@
+import { Money } from '@commercetools-test-data/commons';
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TRANSACTION_STATE, TRANSACTION_TYPE } from '../constants';
+import { TTransactionDraft } from '../types';
+
+const [getOlderDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/payments#transactiondraft
+
+const generator = Generator<TTransactionDraft>({
+  fields: {
+    timestamp: fake(getOlderDate),
+    type: oneOf(...Object.values(TRANSACTION_TYPE)),
+    amount: fake((f) => Money.random()),
```

Comment:
Same here, `CentPrecisionMoney`?

Reply:
Oops - I changed it and then noticed that this one should actually be the `Money` model...

Reply:
Good catch! `Money` it is

Reply:
Huh so I wonder if the pattern is `CentPrecisionMoney` for representation, `Money` for draft